sdk/openai/Azure.OpenAI.Inference/src/Generated/Models/CompletionsRequest.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.OpenAI.Inference.Models
+{
+    /// <summary> The CompletionsRequest. </summary>
+    public partial class CompletionsRequest
+    {
+        /// <summary> Initializes a new instance of CompletionsRequest. </summary>
+        public CompletionsRequest()
+        {
+            Prompt = new List<string>();
+            Logit_bias = new ChangeTrackingDictionary<string, int>();
+            Stop = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Gets the prompt. </summary>
+        public IList<string> Prompt { get; }
+        /// <summary> Gets or sets the max tokens. </summary>
+        public int? Max_tokens { get; set; }
+        /// <summary> Gets or sets the temperature. </summary>
+        public float? Temperature { get; set; }
+        /// <summary> Gets or sets the top p. </summary>
+        public float? Top_p { get; set; }
```

Comment:
Do we plan on renaming these properties as part of this PR?

Reply:
archboard review will create a lot of cadl changes we just generate what the cadl says